### PR TITLE
lesson 02 flow control

### PR DESCRIPTION
**Pull request by AxelNowaczyk**
@tomekDiakowTieto

### CODE REVIEW

File: students/axel_nowaczyk/lesson_02_flow_control/Factorial.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def factorial():
+    number = int(input())
+    product = 1
+    for i in range(2, number+1):
```
Comment (tomekDiakowTieto) [inline students/axel_nowaczyk/lesson_02_flow_control/Factorial.py]: Good

File: students/axel_nowaczyk/lesson_02_flow_control/The_number_of_elements_equal_to_the_maximum.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+def number_of_elements_equal_to_the_maximum():
+    inputs = []
+    while True:
```
Comment (tomekDiakowTieto) [inline students/axel_nowaczyk/lesson_02_flow_control/The_number_of_elements_equal_to_the_maximum.py]: Try to use other condition (without True and break)

File: students/axel_nowaczyk/lesson_02_flow_control/The_number_of_elements_equal_to_the_maximum.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        number = int(input())
+        if number == 0:
+            break
+        inputs.append(number)
+    sorted_inputs = sorted(inputs, reverse=True)
+    max = sorted_inputs[0]
```
Comment (tomekDiakowTieto) [inline students/axel_nowaczyk/lesson_02_flow_control/The_number_of_elements_equal_to_the_maximum.py]: max is built in function. Be carefull:
>>> max
<built-in function max>
>>> max([1,5,2])
5
>>> max = 20
>>> max([1,5,2])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not callable
>>>

File: students/axel_nowaczyk/lesson_02_flow_control/The_number_of_elements_equal_to_the_maximum.py (hunk 2 above)
Comment (AxelNowaczyk) [reply students/axel_nowaczyk/lesson_02_flow_control/The_number_of_elements_equal_to_the_maximum.py]: Similar condition I have found on official wiki.python.org website as common practice. Link:  https://wiki.python.org/moin/WhileLoop

**Pull request merged**